app/queries/cohort_analytics_query.rb

```diff
@@ -33,26 +33,26 @@ def patient_counts_by_period(period, prev_periods, from_time: Time.current)
   end
 
   def patient_counts(cohort_start, cohort_end, report_start, report_end)
-    registered_patients = registered(cohort_start, cohort_end)
-    followed_up_patients = followed_up(registered_patients, report_start, report_end)
+    cohort_patients = registered(cohort_start, cohort_end)
+    followed_up_patients = followed_up(cohort_patients, report_start, report_end)
     controlled_patients = controlled(followed_up_patients)
     uncontrolled_patients = followed_up_patients - controlled_patients
 
-    registered_counts = registered_patients.group(:registration_facility_id).size.symbolize_keys
-    followed_up_counts = followed_up_patients.group(:registration_facility_id).size.symbolize_keys
-    defaulted_counts = registered_counts.merge(followed_up_counts) { |_, registered, followed_up|
-      registered - followed_up
+    cohort_patient_counts = cohort_patients.group(:assigned_facility_id).size.symbolize_keys
+    followed_up_counts = followed_up_patients.group(:assigned_facility_id).size.symbolize_keys
+    defaulted_counts = cohort_patient_counts.merge(followed_up_counts) { |_, cohort_patients, followed_up|
+      cohort_patients - followed_up
     }
 
-    controlled_counts = controlled_patients.group(:registration_facility_id).size.symbolize_keys
+    controlled_counts = controlled_patients.group(:assigned_facility_id).size.symbolize_keys
     uncontrolled_counts = followed_up_counts.merge(controlled_counts) { |_, followed_up, controlled|
       followed_up - controlled
     }
 
     {
-      registered: {total: registered_patients.size, **registered_counts},
+      cohort_patients: {total: cohort_patients.size, **cohort_patient_counts},
```

Comment:
Is this payload cached somewhere? If so, one pattern I've grown fond of is introducing a version number in the cache key. Schema changes like this can be accompanied by bump in the version number, so we don't have to go clear caches as soon as we deploy changes like this.

[Example](https://github.com/simpledotorg/simple-server/blob/progress-control-01/app/services/control_rate_service.rb#L2).

Reply:
This is [cached here](https://github.com/simpledotorg/simple-server/blob/23c26d04ef39c41db53d0aa44132911436865e99/app/controllers/analytics_controller.rb#L50). Added the version number in e4f26d6.